ui/src/views/compute/backup/FormSchedule.vue

```diff
@@ -173,7 +173,8 @@ export default {
       timeZoneMap: [],
       fetching: false,
       actionLoading: false,
-      listDayOfWeek: ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday']
+      listDayOfWeek: ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'],
+      value1: ''
```

Comment:
@utchoang `value1` used anywhere?

Reply:
@sureshanaparti removed it after rebase to 4.17 branch